docs/HONEYBADGER_KO.md

```diff
@@ -0,0 +1,193 @@
+# HoneyBadger BFT Protocol
+### Introduction
+기존의 분산 장애 허용 프로토콜(Distributed fault tolerance protocol)은 주로 작은 규모, 단일 관리 도메인(Single administrative domain)을 
+목표로 배포되고 있었습니다.<br>
+그러나 최근에는 새로운 분산 시스템인 *암호화폐(cryptocurrencies)*, *블록체인(blockchains)* 이 등장하면서, 예측이 힘든 큰 규모의 네트워크에 적합하며, 
+악의적인 노드들에 저항성을 갖는 시스템들에 대한 연구들이 진행되었습니다.<br>
+이런 프로토콜은 아래와 같은 점들을 만족해야 합니다.<br>
+* **견고함**<br>
```

Comment:
1. It is better to write it in English because the Korean is awkward. like a `견고함` or `지연 대비 처리량 향상`

2. These descriptions are only for blockchain. Is not HBBFT an algorithm for blockchain only?